packages/gestalt/src/Tag.css

```diff
@@ -6,11 +6,11 @@
 }
 
 html[dir="rtl"] .button {
-  border-radius: 8px 0 0 8px;
+  border-radius: var(--rounding-200) 0 0 var(--rounding-200);
```

Comment:
could we tokenize this as alias? ... just wondering. 

rounding_container_rtl
rounding_container_ltr